app/javascript/components/widgets/widgets/forest-change/tree-cover-gain/selectors.js

```diff
@@ -76,16 +77,24 @@ export const parseData = createSelector(
 );
 
 export const getSentence = createSelector(
-  [getSortedData, getSettings, getIndicator, getCurrentLocation, getSentences],
-  (data, settings, indicator, currentLabel, sentences) => {
+  [
+    getSortedData,
+    getSettings,
+    getIndicator,
+    getCurrentLocation,
+    getCurrentLabel,
+    getSentences
+  ],
+  (data, settings, indicator, currentLocation, currentLabel, sentences) => {
     if (!data || !data.length) return null;
     const {
       initial,
       withIndicator,
       globalWithIndicator,
       globalInitial
     } = sentences;
-    const locationData = currentLabel && data.find(l => l.id === currentLabel);
+    const locationData =
+      currentLabel && data.find(l => l.id === currentLocation.value);
```

Comment:
you are using currentLabel for checking. This should be the same as the comparison at the end of the line.